ngraph/core/include/ngraph/op/unsqueeze.hpp

```diff
@@ -55,5 +50,3 @@ namespace ngraph
         using v0::Unsqueeze;
```

Comment:
please, remove this line and update places where `op::Unsqueeze` is used instead of `op::v0::Unsqueeze`.

@ilyachur , am I right?

Reply:
Yes you are right!

Reply:
Done.